include/freerdp/codec/color.h

```diff
@@ -69,87 +60,68 @@
  */
 
 /* 32bpp formats */
-#define PIXEL_FORMAT_A8R8G8B8_F(_flip)	FREERDP_PIXEL_FORMAT(_flip, 32, FREERDP_PIXEL_FORMAT_TYPE_ARGB, 8, 8, 8, 8)
+#define PIXEL_FORMAT_A8R8G8B8_F()	FREERDP_PIXEL_FORMAT(32, FREERDP_PIXEL_FORMAT_TYPE_ARGB, 8, 8, 8, 8)
```

Comment:
I think we can get rid of all the _F macros and just use FREERDP_PIXEL_FORMAT directly for the #defines.